generators/server/templates/gradle/kotlin.gradle.ejs

```diff
@@ -51,7 +52,19 @@ dependencies {
 
     testImplementation "org.jetbrains.kotlin:kotlin-test-junit:${kotlin_version}"
 
-    testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:<%= MOCKITO_KOTLIN_VERSION %>"
+    testImplementation "org.mockito.kotlin:mockito-kotlin:<%= MOCKITO_KOTLIN_VERSION %>"
+
+    /*
+     * Latest Detekt has dependencies on jcenter(). This ensures that mavenCentral() can be used to avoid surprises
+     * on 2020-02-01 when jcenter() shuts down. 
```

Comment:
is this date correct?

Reply:
No - it should be 2022

Reply:
No - it should be 2022. I'll fix this along with a few other things over the weekend.